src/fillTank.test.js

```diff
@@ -1,11 +1,48 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+let customer;
 
+  beforeEach(() => {
+    customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
   });
 
-  // write tests here
+  it('should fill tank fully if no amount provided', () => {
+    fillTank(customer, 50);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+    expect(customer.money).toBe(1400);
+  });
+
+  it(`should not exceed tank capacity if more fuel requested`, () => {
+    fillTank(customer, 50, 50);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+    expect(customer.money).toBe(1400);
```

Comment:
In the test case 'should not exceed tank capacity if more fuel requested', you are passing 50 as the third argument to `fillTank`. However, this does not accurately test the scenario described, as the tank capacity is 40. You should pass a value greater than 40 to accurately test this scenario.